roles/pulibrary.rvm/README.md

```diff
@@ -0,0 +1,166 @@
+Role Name
+=========
+
+Install and manage ruby versions using rvm. Bootstrapped from
+[https://github.com/rvm/rvm1-ansible](https://github.com/rvm/rvm1-ansible)
+
+
+Requirements
+------------
+
+Ubuntu Bionic
+
+Role Variables
+--------------
+
+Below is a list of default values that you can configure:
+
+```yaml
+---
+
+# Install 1 or more versions of ruby
+# The last ruby listed will be set as the default ruby
+rvm1_rubies:
+  - 'ruby-2.4.6'
```

Comment:
This should definitely improve things, as a well as permit installations of JRuby.